website/tests/e2e.fixture.ts

```diff
@@ -91,8 +93,36 @@ export const testSequenceCount: number =
 
 const testUserTokens: Record<string, TokenCookie> = {};
 
+async function ensureKeycloakIsReachable(issuerUrl: string) {
+    try {
+        const response = await axios.get(issuerUrl);
+        if (response.status === 200) {
+            e2eLogger.debug(`Server reachable at ${issuerUrl}.`);
+        } else {
+            e2eLogger.warn(`Server responded with status ${response.status} at ${issuerUrl}.`);
+        }
+    } catch (error) {
+        /* eslint-disable no-console */
+        e2eLogger.error(`Failed to reach server at ${issuerUrl}: ${error}`);
+        if (axios.isAxiosError(error)) {
+            console.error('Axios error message:', error.message);
```

Comment:
Should we just use the e2elogger?

Reply:
this is likely resolved now? @fhennig

Reply:
My suggestion was not to use `console.log`

Reply:
(one reason is it can be nice to be able to search all files for `console.log` after dev work to find the ones you added)

Reply:
yes I have removed the console log https://github.com/loculus-project/loculus/pull/2995/commits/2e17caacd9d52d7427521ab1f914624a23c857db